src/app/components/Toggle/Toggle.tsx

```diff
@@ -6,11 +6,15 @@ const Input = styled.input`
 `;
 
 const Handle = styled.div`
-	${tw`inline-flex rounded-full relative h-2 w-10 cursor-pointer bg-theme-neutral-light`}
+	height: 7px;
```

Comment:
Maybe use `rem` instead of `px` :hmhm:? Tailwind uses `rem` for all types of sizes.

Reply:
I would say using px here is fine since tailwind doesn't have a size for 7px, and using it once doesn't warrant it being added to the tailwind config 🤔

Reply:
You're right! :+1:

Reply:
Indeed, 7px seemed just the right size to make the placement of the handle align with the reduced  gray line and as Alex mentioned it wasn't possible with tw classes.

Thanks for reviewing!